### PR TITLE
[ImageBuilder] Upgrade API verision from 2020-02-14 to 2022-02-14

### PR DESCRIPTION
**Pull request by BethanyZhou**
## Description
Move https://github.com/Azure/azure-powershell/pull/19156 to main

## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Comment by isra-fel** [comment]
Please select the checkbox and suppress signature issues. Thanks

File: src/ImageBuilder/examples/Remove-AzImageBuilderTemplate.md [hunk 1]
```diff
@@ -1,16 +1,14 @@
 ### Example 1: Remove a image template
 ```powershell
-PS C:\> Remove-AzImageBuilderTemplate -ImageTemplateName template-name-dmt6ze -ResourceGroupName wyunchi-imagebuilder
-
+Remove-AzImageBuilderTemplate -Name bez-test-img-temp13 
+-ResourceGroupName bez-rg
```
Comment (BethanyZhou) [inline src/ImageBuilder/examples/Remove-AzImageBuilderTemplate.md:4]:
```suggestion
Remove-AzImageBuilderTemplate -Name bez-test-img-temp13 -ResourceGroupName bez-rg
```

File: src/ImageBuilder/help/Remove-AzImageBuilderTemplate.md [hunk 2]
```diff
@@ -32,15 +31,15 @@ Delete a virtual machine image template
 
 ### Example 1: Remove a image template
 ```powershell
-Remove-AzImageBuilderTemplate -ImageTemplateName template-name-dmt6ze -ResourceGroupName wyunchi-imagebuilder
+Remove-AzImageBuilderTemplate -Name bez-test-img-temp13 
+-ResourceGroupName bez-rg
```
Comment (BethanyZhou) [inline src/ImageBuilder/help/Remove-AzImageBuilderTemplate.md:35]:
```suggestion
Remove-AzImageBuilderTemplate -Name bez-test-img-temp13 -ResourceGroupName bez-rg
```

File: src/ImageBuilder/exports/Remove-AzImageBuilderTemplate.ps1 [hunk 3]
```diff
@@ -19,12 +20,10 @@ Delete a virtual machine image template
 .Description
 Delete a virtual machine image template
 .Example
-PS C:\> Remove-AzImageBuilderTemplate -ImageTemplateName template-name-dmt6ze -ResourceGroupName wyunchi-imagebuilder
-
+Remove-AzImageBuilderTemplate -Name bez-test-img-temp13 
+-ResourceGroupName bez-rg
```
Comment (BethanyZhou) [inline src/ImageBuilder/exports/Remove-AzImageBuilderTemplate.ps1:24]:
```suggestion
Remove-AzImageBuilderTemplate -Name bez-test-img-temp13 -ResourceGroupName bez-rg
```

**Comment by BethanyZhou** [comment]
Fix example issues in generation branch by https://github.com/Azure/azure-powershell/pull/19314

**Review by isra-fel — APPROVED** [review]

**Pull request merged**